src/styles/cards.scss

```
@@ -0,0 +1,121 @@
+.container {
+  --columns-count: 1;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(var(--columns-count), 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --columns-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --columns-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --columns-count: 4;
+  }
+}
+
```

You can omit when use scss
```suggestion
  padding: 50px 40px;
  gap: 46px 48px;
  @media (min-width: 488px) {
    --columns-count: 2;
   }
  @media (min-width: 768px) {
     --columns-count: 3;
   }
   @media (min-width: 1024px) {
     --columns-count: 4;
    }
}

```